pkg/rancher-desktop/components/Snapshots.vue

```diff
@@ -0,0 +1,61 @@
+<script lang="ts">
+import Vue from 'vue';
+import { mapGetters } from 'vuex';
+
+import SnapshotCard from '@pkg/components/SnapshotCard.vue';
+
+interface Data {
+  // eslint-disable-next-line no-undef
+  snapshotsPollingInterval: NodeJS.Timer | null,
+}
+
+export default Vue.extend<Data, any, any, any>({
```

Comment:
We can make use of a `VuexBindings` interface to get around relying on `any`

```suggestion
interface VuexBindings {
  snapshots: Snapshot[]
}

export default (Vue as VueConstructor<Vue & VuexBindings>).extend({
```

We have some examples peppered throughout the code. Take a look at for example `pkg/rancher-desktop/pages/Preferences.vue`